### DISCUSSION TITLE
How to make validation error red border only around the TextBox?

### DISCUSSION THREAD

**Discussion by GF-Huang** [HandyOrg/HandyControl#749] [Q&A] [answered]
When there's validation error, the red border around the `TitleElement.Title`, how to make it only around the `TextBox`.

![image](https://user-images.githubusercontent.com/4510984/112685593-c88c7000-8eaf-11eb-9b10-62945dd8ce45.png)

![image](https://user-images.githubusercontent.com/4510984/112685614-cf1ae780-8eaf-11eb-8f1c-2acbffeb7bb5.png)

**Comment by NaBian** [comment, marked as answer]
fixed in b7663e3e758ababf9d3bdd3dfae5cee91d95c413
